test/unit/alignment/pairwise/fixture/global_affine_unbanded.hpp

```diff
@@ -135,4 +135,255 @@ static auto dna4_02 = []()
+        /*A*/ u  ,DuL,uL ,DuL,DuL,Dul,uL ,DuL,uL ,DuL,Dul,uL ,ul ,uL ,DuL,DuL,ul ,ul ,DuL,uL ,uL ,DUL,DuL,l  ,Ul ,Dul,L  ,l  ,
+        /*C*/ u  ,uL ,uL ,uL ,DuL,Dul,uL ,uL ,DuL,DuL,ul ,DuL,DuL,uL ,uL ,DuL,Dul,uL ,ul ,DuL,uL ,DuL,DUl,DuL,L  ,Ul ,DUl,DUl,
+        /*G*/ u  ,uL ,DuL,uL ,DuL,ul ,DuL,uL ,uL ,DuL,ul ,DuL,uL ,DuL,uL ,DuL,uL ,ul ,ul ,uL ,DuL,DuL,Dul,DUL,Dul,uL ,DUl,DUL,
+        /*A*/ u  ,DuL,uL ,DuL,DuL,ul ,uL ,DuL,uL ,DuL,Dul,uL ,uL ,uL ,DuL,DuL,Dul,uL ,Dul,uL ,uL ,DuL,L  ,Dul,DUl,Dul,L  ,Dul
+        }
+    };
+}();
+
+static auto dna4_match_4_mismatch_5_gap_1_open_10_part_04 = []()
+{
+    return alignment_fixture
+    {
+        "AGATCGACTAGCG"_dna4,
+        "G"_dna4,
+        align_config | align_cfg::scoring{nucleotide_scoring_scheme{match_score{4}, mismatch_score{-5}}},
+        -18,
+        "AGATCGACTAGCG",
+        "------------G",
+        alignment_coordinate{column_index_type{0u}, row_index_type{0u}},
+        alignment_coordinate{column_index_type{13u}, row_index_type{1u}},
+        std::vector
+        {
+        //      e,  A,  G,  A,  T,  C,  G,  A,  C,  T,  A,  G,  C,  G,
+        /*e*/ 0  ,-11,-12,-13,-14,-15,-16,-17,-18,-19,-20,-21,-22,-23,
+        /*G*/ -11,-5 ,-7 ,-17,-18,-19,-11,-21,-22,-23,-24,-16,-26,-18
+        },
+        std::vector
+        {
+        //      e,  A,  G,  A,  T,  C,  G,  A,  C,  T,  A,  G,  C,  G,
+        /*e*/ N  ,L  ,l  ,l  ,l  ,l  ,l  ,l  ,l  ,l  ,l  ,l  ,l  ,l  ,
+        /*G*/ U  ,DUL,DUL,DUl,DUl,DUl,DUl,DUl,DUl,DUl,DUl,DUl,DUl,DUl
+        }
+    };
+}();
+
+static auto dna4_match_4_mismatch_5_gap_1_open_10_part_05 = []()
+{
+    return alignment_fixture
+    {
+        "A"_dna4,
+        "AGATCGACTAGCG"_dna4,
+        align_config | align_cfg::scoring{nucleotide_scoring_scheme{match_score{4}, mismatch_score{-5}}},
+        -18,
+        "A------------",
+        "AGATCGACTAGCG",
+        alignment_coordinate{column_index_type{0u}, row_index_type{0u}},
+        alignment_coordinate{column_index_type{1u}, row_index_type{13u}},
+        std::vector
+        {
+        //       e,  A,
+        /*e*/  0  ,-11,
+        /*A*/  -11,4  ,
+        /*G*/  -12,-7 ,
+        /*A*/  -13,-8 ,
+        /*T*/  -14,-9 ,
+        /*C*/  -15,-10,
+        /*G*/  -16,-11,
+        /*A*/  -17,-12,
+        /*C*/  -18,-13,
+        /*T*/  -19,-14,
+        /*A*/  -20,-15,
+        /*G*/  -21,-16,
+        /*C*/  -22,-17,
+        /*G*/  -23,-18
+        },
+        std::vector
+        {
+        //       e,  A,
+        /*e*/  N  ,L  ,
+        /*A*/  U  ,DUL,
+        /*G*/  u  ,UL ,
+        /*A*/  u  ,DuL,
+        /*T*/  u  ,uL ,
+        /*C*/  u  ,uL ,
+        /*G*/  u  ,uL ,
+        /*A*/  u  ,DuL,
+        /*C*/  u  ,uL ,
+        /*T*/  u  ,uL ,
+        /*A*/  u  ,DuL,
+        /*G*/  u  ,uL ,
+        /*C*/  u  ,uL ,
+        /*G*/  u  ,uL
+        }
+    };
+}();
+
+
+static auto dna4_match_4_mismatch_5_gap_1_open_10_seq1_empty_part_06 = []()
```

Comment:
Do you want to have the `part_06` part here (and the next two)? The `seq_empty` makes it already different from the others before.

Reply:
probably not. I did wanted to know that they are special because of empty sequences.

Reply:
Do as you like